glide.lock

```diff
@@ -286,6 +279,10 @@ testImports:
   version: d8ed2627bdf02c080bf22230dbb337003b7aba2d
   subpackages:
   - difflib
+- name: github.com/sirupsen/logrus
```

Comment:
@doodles526  there's already `sirupsen/logrus` at line 102 and this entry is probably making travis unhappy.

Also, not sure how big of a deal that is, but logrus got bumped (I dunno if we locked in the semver in glide.yaml so maybe that's a big bump?)

Reply:
It's locked onto 1.0.3. So not a big bump. Getting rid of that ref and seeing if it passes now